src/DKCTropicalFreeze/Cheats/DKC_BananaMult/rules.txt

```diff
@@ -0,0 +1,30 @@
+[Definition]
+titleIds = 0005000010137F00,0005000010138300,0005000010144800
+name = Banana Multiplier
+path = "Donkey Kong Country: Tropical Freeze/Cheats/Banana Multiplier"
+description = Multiplies the amount of bananas you collect.
+#Credits: Intra
+version = 7
+
+[Default]
+$addamountbna = 1
+
```

Comment:
```suggestion

[Preset]
name = x1
default = 1

```
It should also have a default preset, maybe?

Reply:
I tested a x1 multiplier for bananas and it makes the 5 banana bundle give you 4 which is not intended behavior for a default x1 setting. (at x2 they give 8 instead of 10, at x3, 11 instead of 15, etc.)
so I dont think it should be added for the banana multiplier because it isn't truly a x1

the x1 option works fine for lives and coins, but I would like to keep the 3 mods consistent with each other and given that I don't think a x1 would be a good idea for the banana mod code wise, I dont want to add a x1 for lives and coins to keep things the same style wise.